lib/data/external/AwsClient.js

```diff
@@ -355,6 +357,35 @@ class AwsClient {
             return callback();
         });
     }
+    copyObject(request, locationType, objectGetInfo, log,
```

Comment:
this is trivial, but I think the name of the second param should be 'location' instead of 'locationType' because it will be the name of the actual location constraint instead of like aws or azure

Reply:
you are right, thanks!